<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>725</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vlay_main">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="fr_main">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlay_fr_main">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_menu_top">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="hlay_menu_top">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_mt_ico"/>
         </item>
         <item>
          <widget class="QWidget" name="wg_mt_blank" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lb_mt_nom"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_mt_option"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_mt_reduire"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_mt_agrandir"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_mt_quitter"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_body">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="fr_menu_titre">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lb_title">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QListWidget" name="list_menu">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_logiciel">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>574</width>
                    <height>637</height>
                   </rect>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QGridLayout" name="gridLayout"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_helper"/>
             <widget class="QWidget" name="page_editeur">
              <widget class="QPushButton" name="pb_addApp_edit">
               <property name="geometry">
                <rect>
                 <x>170</x>
                 <y>210</y>
                 <width>101</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="text">
                <string>Ajout_app</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_admin">
              <widget class="QPushButton" name="pb_addApp_admin">
               <property name="geometry">
                <rect>
                 <x>230</x>
                 <y>230</y>
                 <width>101</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="text">
                <string>Ajout_app</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_ajout_app">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="DropLabel" name="lb_icon_new">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>232</width>
                        <height>232</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>232</width>
                        <height>232</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">.QLabel{
background-color: rgb(85, 255, 0);
}</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pb_ico">
                        <property name="text">
                         <string>Icon</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="8" column="1" colspan="3">
                     <widget class="QPushButton" name="pb_new_ap_val">
                      <property name="text">
                       <string>Enregistrer</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="3">
                     <spacer name="horizontalSpacer_6">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="2" column="3">
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="5" column="1" colspan="4">
                     <widget class="QTextEdit" name="text_desc">
                      <property name="undoRedoEnabled">
                       <bool>true</bool>
                      </property>
                      <property name="markdown">
                       <string/>
                      </property>
                      <property name="html">
                       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                      <property name="tabStopDistance">
                       <double>20.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1" colspan="2">
                     <widget class="QDoubleSpinBox" name="num_ver">
                      <property name="minimum">
                       <double>0.010000000000000</double>
                      </property>
                      <property name="maximum">
                       <double>99999999.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.010000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1" colspan="2">
                     <widget class="QComboBox" name="cb_auteur"/>
                    </item>
                    <item row="0" column="1" colspan="4">
                     <widget class="QLineEdit" name="line_ap"/>
                    </item>
                    <item row="7" column="1" colspan="4">
                     <widget class="QTextEdit" name="text_chang"/>
                    </item>
                    <item row="5" column="0">
                     <widget class="QLabel" name="lb_desc_ap">
                      <property name="text">
                       <string>Description</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="lb_auteur_ap">
                      <property name="text">
                       <string>Auteur</string>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="0">
                     <widget class="QLabel" name="lb_chang_ap">
                      <property name="text">
                       <string>Correctif</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="lb_ver_ap">
                      <property name="text">
                       <string>Version</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QPushButton" name="pb_ap_selector">
                      <property name="text">
                       <string>Application</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="lb_nom_ap">
                      <property name="text">
                       <string>Nom</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1" colspan="4">
                     <widget class="QLineEdit" name="line_ap_nom"/>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="lb_stock">
                      <property name="text">
                       <string>Stokage</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1" colspan="3">
                     <widget class="QComboBox" name="cb_stock"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_menu_bottom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="hlay_menu_bottom">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_mb_version"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropLabel</class>
   <extends>QLabel</extends>
   <header>src.widget_custom.Label_cust</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
